templates/table.html.tera

```diff
@@ -60,6 +69,17 @@
     </div>
 </nav>
 <div class="container-fluid">
+    {% if description %}
+    <div class="row" style="margin-top: 15px;">
+        <div class="col-md-12">
+            <div class="collapse" id="collapseDescription">
+                <div class="card card-body">
+                    {{ description }}
```

Comment:
Should be allowed to have markdown in the description, which one should then render in JS here.

Reply:
Good idea! Implemented this in 24d9393.